engine/src/main/java/com/datatorrent/stram/engine/OperatorContext.java

```diff
@@ -84,12 +87,13 @@ public void setIdleTimeout(long idleTimeout)
    * @param attributes the value of attributes
    * @param parentContext
    */
-  public OperatorContext(int id, AttributeMap attributes, Context parentContext)
+  public OperatorContext(int id, String operatorName, AttributeMap attributes, Context parentContext)
```

Comment:
id, name and attributes seem to always come from OperatorDeployInfo. If this is the case, should OperatorContext take OperatorDeployInfo as input parameter instead?